lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -240,4 +245,38 @@ class DSCard extends StatelessWidget {
       );
     }
   }
+
+  Widget _buildRequestLocation() {
```

Comment:
I'd like to suggest to you create a widget called `DSRequestLocationMessageBubble` and create the logic of this function inside of it, passing the params just as it was done on `DSWeblink`.